controllers/HomeController.php

```diff
@@ -10,5 +10,35 @@
 
 class HomeController extends BaseController
 {
+    private static $name = NULL;
+    private static $email = NULL;
+    private static $message = NULL;
+    private static $token = NULL;
 
+    public function __construct()
+    {
+        $app = new Factory;
+        $mysql = $app->getDBO();
+
+        $request = new Request();
+        
+        self::$name = $request->get('name');
+        self::$email = $request->get('email');
+        self::$message = $request->get('message');
+    }
+
+    public static function sendFeedback()
+    {
+        $db = new Factory;
+        $mysql = $db->getDBO();
+    
+        $sql = "INSERT INTO feedback (name,email,message) VALUES ('$name','$email','$message')";
```

Comment:
`$name`, `$email` and `$message ` is undefined because you saving the values in static `self::$name`. 
So, add before this line `$name = self::$name`